src/app/modules/users/single-user/single-user.component.html

```diff
@@ -14,26 +14,31 @@ <h3 class="ftitle">User Creation</h3>
         </div>
 
         <div class="col-md-4">
-            <div *ngIf="loading" class="modal-body">
+            <app-dynamic-form *ngIf="loading" [questions]="questions"></app-dynamic-form>
```

Comment:
Where is this component defined?

Reply:
In shared module

Reply:
Currently , i replaced with other dynamic forms